client/payment-request/blocks/use-initialization.js

```diff
@@ -87,19 +95,6 @@ export const useInitialization = ( {
 		billing.cartTotalItems,
 	] );
 
-	const abortPayment = ( paymentMethod, message ) => {
```

Comment:
Moved their definition inside the `useEffect` hook - otherwise they would need to be declared as dependency and "stabilized" through `useCallback`